tox_poetry_installer/installer.py

```diff
@@ -9,8 +9,10 @@
 from typing import Collection
 from typing import Set
 
+from poetry.config.config import Config
 from poetry.core.packages.package import Package as PoetryPackage
-from tox.venv import VirtualEnv as ToxVirtualEnv
+from poetry.installation.operations.install import Install
```

Comment:
The `from poetry...` imports in this file need to be proxied through the `tox_poetry_installer._poetry` module. By importing them directly in the top level it'll cause an import error (specifically a `ModuleNotFoundError`) when running Tox if `poetry` isn't installed.

Reply:
Oh I missed that, thanks for the hint! It's fixed now.